presto-orc/src/test/java/com/facebook/presto/orc/stream/TestLongStreamDwrfAriaScan.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.stream;
+
+import com.facebook.presto.orc.OrcCorruptionException;
+import com.facebook.presto.orc.OrcDecompressor;
+import io.airlift.slice.Slice;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+
+import static com.facebook.presto.memory.context.AggregatedMemoryContext.newSimpleAggregatedMemoryContext;
+import static com.facebook.presto.orc.OrcDecompressor.createOrcDecompressor;
+import static com.facebook.presto.orc.metadata.CompressionKind.SNAPPY;
+import static com.facebook.presto.orc.metadata.OrcType.OrcTypeKind.LONG;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.DATA;
+
+public class TestLongStreamDwrfAriaScan
+        extends AbstractTestValueStreamAriaScan<LongOutputStreamDwrf, LongInputStreamDwrf>
+{
+    @Test
+    public void test()
+            throws IOException
+    {
+        testScan(getTestValues());
+    }
+
+    private List<List<Long>> getTestValues()
+    {
+        List<List<Long>> groups = new ArrayList<>();
```

Comment:
@elonazoulay I expected this logic to be a copy-paste from `com.facebook.presto.orc.stream.TestLongStreamDwrf`, but it is different. Would you explain why?

Reply:
The data set from TestLongInputStreamV1 covers more cases: 
* It has multiple checkouts
* Sequentially increasing data
* Random data

The test is similar to TestLongStreamV1AriaScan.

Was there anything specific you wanted to know?

Reply:
@elonazoulay I looked at existing tests `TestLongStreamV1` and `TestLongStreamDwrf`. These use different groups. However, new tests, which I expect to be essentially the same but go through Aria path, have the same groups. I think these should be consistent. Perhaps `TestLongStreamDwrf` needs to be updated to use more interesting dataset. 

Now, AriaV1 and AriaDwrf use same groups. The only remaining test to add, AriaV2, will use same groups as well. No other tests are possible because base class calls into `scan` method that's available only for long type. Hence, it would make sense to move `getTestValues` method into the base class.

Reply:
Just to be thorough I added this as another test.

Reply:
@elonazoulay Elon, 

> Just to be thorough I added this as another test.

I'm not quite sure I understand what you are saying here. Do you think it makes sense to move `getTestValues` to the base class?

Reply:
To clarify: I added the test data from TestLongInputStreamDwrf to the TestLongInputStreamDwrfAriaScan so that it has 2 tests.

Reply:
@elonazoulay I don't see any reason to use different datasets for v1, v2 and dwrf. It would be simpler to use same datasets for all 3 and have these defined in the base class.